### PR TITLE
Feat/readme.md

### LINKED ISSUE #45: Add example videos to Readme

### PR DESCRIPTION
**Pull request by georgbuechner**
- Adds installation instructions 
- Adds example images 
- Adds outline 
- Fixes #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,146 @@
...
 ```
 litt <index-name> <search-term>
 ```
-If your search term is more than one word, you should add quotations: `litt
-<index-name> '<term1 term2 ...>`
+If your search term is more than one word, you should add quotations: `litt <index-name> '<term1 term2 ...>'`
+
+Use `--offset` and `--limit` to show more results. (Default shows the top ten
+results. `--offset 10` shows the first 10 to 20 results. `--limit 50 --offset 10`
+shows the first 10 to 50 results).
```
Comment (SimonThormeyer) [inline README.md:124]: Wouldn't limit 50 and offset 50 mean that it's 10 to 60?

File: README.md [hunk 2]
```diff
@@ -1,45 +1,146 @@
...
+```
+(Replace version and platform accordingly)
+
+**Usage**:
+```
+litt <index-name> -i <path-to-documents>
+litt <index-name> <search-term>
+```
+
+![Title Image](images/title-smaller.gif)
```
Comment (SimonThormeyer) [inline README.md:19]: This is great, but is it possible to record this again, with a larger font size? I can barely see anything on my screen.

Or we could just increase the size of the image, possibly with html if md doesn't support this?

File: README.md [hunk 3]
```diff
@@ -1,45 +1,146 @@
...
+
+#### Windows 
+Honestly, I don't really know. After downloading the file `litt` should be added
+to path. This guide gives some explanation on how to do that: 
+https://windowsloop.com/how-to-add-to-windows-path/
 
 ## Usage 
 
-### Create a new LITT-index 
+![basic functionality](images/basic_functionality.png)
```
Comment (SimonThormeyer) [inline README.md:82]: Same here

File: README.md [hunk 4]
```diff
@@ -1,45 +1,146 @@
...
 ```
-If your search term is more than one word, you should add quotations: `litt
-<index-name> '<term1 term2 ...>`
+If your search term is more than one word, you should add quotations: `litt <index-name> '<term1 term2 ...>'`
+
+Use `--offset` and `--limit` to show more results. (Default shows the top ten
+results. `--offset 10` shows the first 10 to 20 results. `--limit 50 --offset 10`
+shows the first 10 to 50 results).
+
+![offset-limit example](images/offset-limit_example.png)
```
Comment (SimonThormeyer) [inline README.md:126]: Same here

**Review by SimonThormeyer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (georgbuechner) [reply README.md]: Fixes in last commit

File: README.md (hunk 3 above)
Comment (georgbuechner) [reply README.md:82]: Fixed.

File: README.md (hunk 4 above)
Comment (georgbuechner) [reply README.md:126]: Fixed.

File: README.md [hunk 5]
```diff
@@ -1,45 +1,146 @@
...
+
+Use `litt <num>` to open a document (num refers to the number in brackets, f.e.
+`- [1] p. XXX: ...`)
+
+![simple example](images/simple_example.png)
```
[the remark this answers is not in the document]
Comment (georgbuechner) [reply README.md:131]: Fixed.

File: README.md [hunk 6]
```diff
@@ -57,6 +158,8 @@ litt books '"Tulpen Narzisse"~1'
 ```
 which will also match f.e. `Tulpen wie Narzisse`.
 
+![phrase example](images/phrase_example.png)
```
[the remark this answers is not in the document]
Comment (georgbuechner) [reply README.md:161]: Fixed.

**Pull request merged**